packages/core/src/App/Components/Elements/LiveChat/use-livechat.ts

```diff
@@ -101,6 +101,11 @@ const useLiveChat = (has_cookie_account = false) => {
         });
     };
 
+    useEffect(() => {
+        onHistoryChange();
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
@sergei-deriv Any reason for not adding `onHistoryChange` to the `useEffect` dependencies? 🤔

Reply:
@farzin-deriv Just one reason, onHistoryChange function has empty dependency array and this function will be constant throughout the component life cycle